### PR TITLE
Added all sorts of info!

### PR DESCRIPTION
**Pull request by jdfergusson**

### CODE REVIEW

File: pages/index.md [hunk 1]
```diff
@@ -121,6 +129,8 @@ He lives in York with his wife Eliza and their 9 stringed instruments, three dru
 </div>
 </div>
 
+[See what's involved in getting involved >>](/helping/)
```
Comment (jamesdoc) [inline pages/index.md:132]: Should be `»`

File: templates/base.html [hunk 2]
```diff
@@ -89,12 +89,12 @@
         <div class="col col-footer">
           <ul class="compact-links">
 
-            <li><a href="https://content.scriptureunion.org.uk/holidays-events">More holidays</a>
+            <li><a href="https://events.scriptureunion.org.uk/">More holidays</a>
             <li><a href="https://www.instagram.com/livewireshol/" target="blank" rel="noopener">Instagram</a>
             <li><a href="https://github.com/livewires" target="blank" rel="noopener">GitHub</a>
             <li><a href="http://downloads.livewires.org.uk/">Downloads</a>
             <li><a href="http://info.livewires.org.uk/helping/">Join the Team</a>
```
Comment (jamesdoc) [inline templates/base.html:96]: This link can now be wired up `/helping`

File: pages/day.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+social_title: "technical activity week for 12–15s"
+social_image: "/assets/livewires-og.jpg"
+description: "LiveWires is a technical activity week for 12–15s. Start your summer learning about Jesus and technology."
+og_description: "Start your summer with a week learning about Jesus and technology."
+---
+
+# A Day On LiveWires
+
+<div class="columns" style="grid-template-columns: 1fr 2.2fr;">
+<img src="/assets/timetable.png" >
```
Comment (jamesdoc) [inline pages/day.md:12]: Please can this have an `alt` attribute… such as…

`src="…" alt="a graphic showing the breakdown of a day into 15 minute increments"`

**Review by jamesdoc — APPROVED** [review]

File: pages/data.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Use of data on social media and the web
+
+LiveWires has official social media channels on Facebook, Twitter, Instagram and YouTube. We also host this website. We do use photographs from LiveWires to update the LiveWires community and to promote LiveWires. When registering, you have the ability to grant permission for photographs of you to be used for either internal or public purposes. We do not tag people in photos and we will not sell any of our photos to third parties or use them for any commercial enterprise. We can not be held responsible for guests on LiveWires uploading content themselves, although we do discourage this. If you have any concerns about photographs being taken, please contact the leadership team.
+
+## Who to contact if you want to correct your data
+
+If you want to update the data we hold for you, please contact the holiday leaders by email at livewires@scriptureunion.org.uk. By booking through the Scripture Union website, they too will store details relating to your booking. Please refer to their data processing guidelines on their website if you have any queries or concerns.
```
Comment (tjvr) [inline pages/data.md:38]: We should probably make this email address a link!

File: pages/data.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (tjvr) [inline pages/data.md:1]: Is this page linked anywhere?

File: pages/index.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ We've got the knowledge and the gear to teach you everything you need to know. A
 
 Dig into the Bible through short talks, chatting with new friends, and praying together. Ask questions about the Christian faith, and explore things that really matter.
 
-<!--<p>In 2022 we’ll be exploring the whole story of the Bible, seeing more of God’s plan for His people, living in His world, and experiencing His blessing.</p>-->
+In 2023 we’ll be exploring the book of Ruth, and seeing how God turns bitterness into blessing
```
Comment (tjvr) [inline pages/index.md:38]:
```suggestion
In 2023 we’ll be exploring the book of Ruth, and seeing how God turns bitterness into blessing.
```

**Review by tjvr — COMMENTED** [review]
This looks pretty good! Are we nearly ready to be able to get rid of `info.livewires` for good? :sweat_smile:

File: pages/data.md (hunk 5 above)
Comment (jdfergusson) [reply pages/data.md:1]: It's linked in `base.html`, in the footer

**Comment by jdfergusson** [comment]
@tjvr @jamesdoc Responded to all the comments, so a quick re-review would be appreciated!

Once this change goes live I think we can bin `info.livewires` for good!

**Pull request merged**